packages/file-types/zarr/src/AnnDataSource.js

```diff
@@ -136,12 +96,10 @@ export default class AnnDataSource extends ZarrDataSource {
    * @returns {Promise} A promise for a zarr array containing the data.
    */
   loadNumeric(path) {
-    const { store } = this;
-    return openArray({
-      store,
-      path,
-      mode: 'r',
-    }).then(arr => arr.get());
+    const { storeRoot } = this;
+    return storeRoot
+      .then(root => zarrOpen(root.resolve(path), { kind: 'array' }))
+      .then(arr => zarrGet(arr, arr.shape.map(() => null)));
```

Comment:
I'm pretty sure you could just pass `null` or omit the second argument here if you want to read the entire array.

```suggestion
      .then(arr => zarrGet(arr));
```

Reply:
Oh nice, thanks!